build/repo.targets

```diff
@@ -69,6 +70,13 @@
     <CloudScenarios Include="-n MvcDbFortunesRaw --webHost KestrelLibuv $(HtmlJobs) --database PostgreSql" />
     <CloudScenarios Include="-n MvcDbFortunesDapper --webHost KestrelLibuv $(HtmlJobs) --database PostgreSql" />
     <CloudScenarios Include="-n MvcDbFortunesEf --webHost KestrelLibuv $(HtmlJobs) --database PostgreSql" />
+
+    <CloudScenarios Include="-n SignalRBroadcast --properties 'TransportType=WebSockets;HubProtocol=json' $(SignalRJobs)" />
```

Comment:
`TransportType=WebSockets;HubProtocol=json` Do you set these properties in the database as custom columns?

Reply:
Yes